public/js/routes/line/index.js

```diff
@@ -7,62 +10,66 @@ var TripsCollection = require( '../../collections/trips' );
 var LineStatusModel = require( '../../models/line-status' );
 
 var data = require( '../../data' );
-var setTitle = require( '../../lib/set-title' );
 
-function lineOverviewRoute( lineSlug ) {
-  /* jshint validthis: true */
+module.exports = {
+  url: '^/:line(red|orange|blue)',
 
-  // Error out early if the route didn't get a valid line slug
-  var invalidLineSlug = [ 'red', 'orange', 'blue' ].indexOf( lineSlug ) < 0;
+  update: function( opts ) {
+    this.enter( opts );
+  },
```

Comment:
@jmeas You listed the [Update Hook](https://github.com/jmeas/router-comparison#update-hook) as a router component you liked, but I'm not sure I understand how that is valuable. If I need to fetch new data, and render a new view, when I transition from e.g. `/red/place-davis` to `/orange/place-sullivan`, it does not feel intuitive that I need to proxy 'update' through to 'enter' in order to have that change be detected and handled.

I may be doing this all wrong :)